dev/preview/previewctl/pkg/preview/util.go

```diff
@@ -20,6 +21,11 @@ import (
 
 func GetName(branch string) (string, error) {
 	var err error
+
+	if branch == "" {
+		branch = os.Getenv("GITHUB_HEAD_REF")
+	}
```

Comment:
```suggestion
	if v := os.Getenv("GITHUB_ACTIONS"); v != "" && branch == "" {
		branch = os.Getenv("GITHUB_HEAD_REF")
	}
```

Reply:
Rebased on main after approving https://github.com/gitpod-io/gitpod/pull/15985 so this is not longer relevant ☺️